### DISCUSSION TITLE
Inovação em Comunidade: Testando uma CLI para Criação de Termos 🚀

### DISCUSSION THREAD

**Discussion by levxyca** [levxyca/diciotech#176] [Q&A]
Olá, Comunidade Diciotechers 🫰🏻

Estamos atualmente em uma fase de atualização do projeto, onde estamos revisando todos os PR's antigos que temos, por isso, gostaria da ajuda de vocês com esse PR em específico. O PR #112 do @rwietter adiciona uma Interface de Linha de Comando (CLI) para criar novos termos 🤩 O que é uma feature super bacana visando facilitar as adições e crecimento do projeto.

Conseguem me ajudar nessa? Gostaria de ouvir as opiniões de vocês sobre essa feature 🚀 Vou deixar essa discussão viva até sexta-feira 28/06 para darmos andamento 👀

**Comment by levxyca** [comment]
@george-gca @andrepg vocês são pessoas que estão bem ativas aqui no projeto recentemente, gostaria da ajuda de vocês também, se possível, com essa validação 🫰🏻

**Reply by andrepg** [reply]
Pode deixar @levxyca ! Confiro certinho e dou opinião! A ideia é genial. 

Se o colega @rwietter não se opor, no futuro podemos, inclusive, adicionar um _fetch_ para buscar termos via CLI.

**Reply by levxyca** [reply]
@andrepg show demais, obrigada! eu acho essa ideia que você trouxe super bacana também!

**Reply by george-gca** [reply]
@levxyca sexta à noite eu consigo parar pra pegar isso pra ver. Essa semana eu não consegui parar ainda, mas dou uma olhada sem falta.

**Reply by levxyca** [reply]
Perfeito @george-gca! Sem problemas!

**Reply by george-gca** [reply]
Eu tô com uma dúvida, e percebi que pode ser uma boa adição pro projeto. Como eu instalo o ambiente de desenvolvimento pra testar de fato o código da PR? Eu não tenho nem o npm instalado na minha máquina, então seria legal no README ter um tópico ensinando a instalar o ambiente de desenvolvimento.

**Reply by rwietter** [reply]
De fato @george-gca, esqueci de adicionar um documento, irei criar um README em /cli para orientar como baixar, instalar e executar. Mas a princípio, já possui o bundle então tendo o Node.js instalado, seria só rodar um `npm run new`

**Reply by george-gca** [reply]
Mas seria legal ter também algo explicando como instalar o node, ou apontar pra algum tutorial do tipo. Pra pessoas que não são desenvolvedores web (como eu) seria muito útil.
